docs/configuration.md

```diff
@@ -131,6 +131,19 @@ JITSI_PIN_URL=
 
 # optional - configure a Open-Xchange link that can be used to open the calendar room in their calendar application. use the {{id}} and {{folder}} as a template parameter (ex: "http://ox.example.com/appsuite/#app=io.ox/calendar&id={{id}}&folder={{folder}}")
 OPEN_XCHANGE_MEETING_URL_TEMPLATE=
+
+# optional - enables changing of power level of the guest users when they join the room. This case also requires that the user's default power level that be higher then guest user power level and that the bot has rights to change power levels, otherwise no change occurs.
+ENABLE_GUEST_USER_POWER_LEVEL_CHANGE=false
+
+# optional - configure guest user id prefix that is used to match guest users.
+GUEST_USER_ID_PREFIX=guest-
```

Comment:
```suggestion
GUEST_USER_PREFIX=guest-
```

Reply:
It will be then aligned to what we have in our [element-web-guest-module](https://github.com/nordeck/element-web-modules/blob/4187111b4b47e04ce0b0fef5fd67452f2426a2b0/packages/element-web-guest-module/README.md?plain=1#L52).

Reply:
Yes, that is a good point, have change it to how it is done on our module, thanks!